sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -653,6 +653,20 @@ class HiveDDLSuite
     }
   }
 
+  test("CTAS - converted to Data Source Table but lost table properties") {
+    withSQLConf(SQLConf.CONVERT_CTAS.key -> "true") {
+      withTable("t") {
+        sql("CREATE TABLE t TBLPROPERTIES('prop1' = 'c', 'prop2' = 'd') AS SELECT 1 as a, 1 as b")
+        val tableDesc = spark.sessionState.catalog.getTableMetadata(TableIdentifier("t"))
+        assert(tableDesc.properties.get("prop1").isEmpty)
+        assert(tableDesc.properties.get("prop2").isEmpty)
+        assert(tableDesc.storage.properties.get("prop1") == Option("c"))
+        assert(tableDesc.storage.properties.get("prop2") == Option("d"))
```

Comment:
is this what we want? Why do the table properties in Hive serde table should go to storage properties in data source table?

Ideally data source table should have data source options(storage properties) and table properties. Currently we don't support specifying table properties for data source tables, but it doesn't mean we will never do it.